Add-ons/UmbracoForms/Developer/Magic-Strings/index.md

```diff
@@ -52,12 +65,12 @@ The `Umbraco.Forms.Data.StringHelpers` class contains helper methods for parsing
 
 ```csharp
 // Does not parse Record-related magic strings - they are simply removed.
-public static string ParsePlaceHolders(HttpContext context, string value) 
+public static string ParsePlaceHolders(HttpContext context, string value)
 
 // Uses the passed in Record to parse Record-related magic strings
 public static string ParsePlaceHolders(Record record, string value)
 
 public static string ParsePlaceHolders(HttpContext context, Record record, string value)
 ```
 
-There is also a public extension method `ParsePlaceHolders()`extending the `string` object in the `Umbraco.Forms.Core.Extensions` namespace that can be used with anything that you wish to try and replace the above tokens in a string, but it doesn't currently work with Records.
+There is also a public extension method `ParsePlaceHolders()` extending the `string` object in the `Umbraco.Forms.Core.Extensions` namespace. This can be used with to replace the above tokens in a string, but it doesn't currently work with records/From field items.
```

Comment:
```suggestion
There is also a public extension method `ParsePlaceHolders()` extending the `string` object in the `Umbraco.Forms.Core.Extensions` namespace. This can be used to replace the above tokens in a string, but it doesn't currently work with records/From field items.
```